FileIO/readMeshFromFile.cpp

```diff
@@ -44,8 +44,16 @@ MeshLib::Mesh* readMeshFromFile(const std::string &file_name)
 {
 #ifdef USE_PETSC
 	NodePartitionedMeshReader read_pmesh(PETSC_COMM_WORLD);
+
+	std::string file_name_ext;
+	if (BaseLib::hasFileExtension("msh", file_name))
+	   file_name_ext = ".msh";
+	if (BaseLib::hasFileExtension("vtk", file_name))
+	   file_name_ext = ".vtk";
+	if (BaseLib::hasFileExtension("vtu", file_name))
+	   file_name_ext = ".vtu";
```

Comment:
I just assume that the default mesh file is either vtk or vtu, and other mesh file could be OGS5 mesh. I also think about to subtract the substring after the last ``.`` in file_name.